torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -591,7 +590,23 @@ def rotate(
 def pad_image_tensor(
     img: torch.Tensor,
     padding: Union[int, List[int]],
-    fill: Optional[Union[int, float]] = 0,
+    fill: Optional[Union[int, float, List[float]]] = None,
```

Comment:
I know this kernel is private on stable and I agree with the change to ensure consistency. Any BC breakages that can creep upwards due to the change on the default value?

Reply:
`pad_image_tensor` is low-level new API so I think there is no BC to keep. In any case fill=None will be transformed to fill=0 inside https://github.com/pytorch/vision/blob/c0911e31d367eca9600541bd634564c73753abbf/torchvision/transforms/functional_tensor.py#L381-L382